Makefile

```diff
@@ -22,14 +22,16 @@ type=0
 image_base="centos"
 image_tag="7"
 iteration=0
+dist=0
 
 ### function for building
 ### $(1) is name
 ### $(2) is dockerfile filename
 ### $(3) is package type
+### $(4) is distribution type (dist)
 define build
 	docker build -t apache/$(1)-$(3):$(version) --build-arg checkout_v=$(checkout) \
-		-f ./dockerfiles/Dockerfile.$(2).$(3) .
+		-f ./dockerfiles/Dockerfile.$(2).$(3).$(4) .
```

Comment:
CI will fail when dist is empty.

Reply:
I think it is ok to support both apisix-dashboard and apisix-openresty.

Reply:
@EnableAsync yep, the latest commit has added all the three packages support. Please take a look at the new implementation https://github.com/api7/apisix-build-tools/pull/67#issuecomment-896536098. Really thanks for your help.